test/test_prototype_transforms_consistency.py

```diff
@@ -34,6 +34,15 @@
 DEFAULT_MAKE_IMAGES_KWARGS = dict(color_spaces=["RGB"], extra_dims=[(4,)])
 
 
+class NotScriptableArgsKwargs(ArgsKwargs):
```

Comment:
Should we have a separate section for jit tests instead of adding a new class? There's a ton of classes and abstractions in the new test framework already, I think we should avoid adding more

Reply:
As discussed offline, we can leave this out for now and reconsider later if need be